Podfile

```diff
@@ -54,7 +55,7 @@ def shared_with_all_pods
     wordpress_shared
     pod 'CocoaLumberjack', '3.5.2'
     pod 'FormatterKit/TimeIntervalFormatter', '1.8.2'
-    pod 'NSObject-SafeExpectations', '0.0.4'
+    pod 'NSObject-SafeExpectations', '~> 0.0.4'
```

Comment:
This is ok, but why do we need to explicitly add ~>? This should be needed if we want to automatically update to the next patch when we update pods, and since 0.0.4 is the latest version, this should really be needed here? (same for the other parts where ~> was added)

Reply:
I think this should be safe as it should only update to 'compatible' versions (0.0.x) and it make make the process smoother when you need to pull in new versions of our own pods which may rely on the same libraries.